### DISCUSSION TITLE
老哥有考虑做一个安卓版本的咩~

### DISCUSSION THREAD

**Discussion by ghost** [xfangfang/Macast#163] [Ideas]
市面上的投屏软件太臃肿使用也繁琐，大佬考不考虑做一个安卓的，用在电视、车机等等，这类安卓版本低，配置也低的设备😋

**Comment by gxj8885718** [comment]
+1

**Reply by Zhxany** [reply]
安卓使用当贝下载乐播老版本，路由器屏蔽广告网址，也非常轻巧，我在电视盒子上有装
